server/config/environment/development.js

```diff
@@ -17,5 +17,5 @@ module.exports = {
   // For testing verification
   // attendanceVerificationRatio: 1,
 
-  seedDB: true
+  seedDB: false
```

Comment:
Yup. I thought we had false as the default since the grunt seed command was created. It must have gotten switched at some point.